### PR TITLE
fix: #14187 TagsVersionPage break having dot in it's name

### PR DESCRIPTION
**Pull request by harsh-vador**
**Describe your changes:**
Earlier the code used to split the classification Name and get the 0th index but it wasn't required. So fixed it by passing the classification name itself.

Fixes: #14187 

https://github.com/open-metadata/OpenMetadata/assets/58542468/5e298387-3539-41f0-8a56-83d3e9bfb6bc

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/pages/ClassificationVersionPage/ClassificationVersionPage.test.tsx [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+    });
+
+    render(<ErrorPlaceHolder type={ERROR_PLACEHOLDER_TYPE.PERMISSION} />, {
+      wrapper: MemoryRouter,
+    });
+
+    const errorPlaceholder = await screen.findByText('testErrorPlaceHolder');
+
+    expect(errorPlaceholder).toBeInTheDocument();
+  });
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/pages/ClassificationVersionPage/ClassificationVersionPage.test.tsx:148]: @harsh-vador , we need to check if the error placeholder is rendered by the version component or not, here we are just rendering the error placeholder component and checking it.

Can we check if the version component rendered the error placeholder instead?

File: openmetadata-ui/src/main/resources/ui/src/pages/ClassificationVersionPage/ClassificationVersionPage.test.tsx [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+        isVersionView
+        classificationPermissions={DEFAULT_ENTITY_PERMISSION}
+        currentClassification={currentVersionData}
+      />
+    );
+
+    const classificationName = screen.getByText('testClassificationDetails');
+
+    expect(classificationName).toBeInTheDocument();
+  });
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/pages/ClassificationVersionPage/ClassificationVersionPage.test.tsx:167]: Same here we need to check if classification details are rendered by version page or not.

File: openmetadata-ui/src/main/resources/ui/src/pages/TagsPage/TagsPage.tsx [hunk 3]
```diff
@@ -118,11 +118,15 @@ const TagsPage = () => {
...
-      setClassificationPermissions(response);
+      if (currentClassification?.id) {
+        const response = await getEntityPermission(
+          ResourceEntity.CLASSIFICATION,
+          currentClassification.id as string
+        );
+        setClassificationPermissions(response);
+      } else {
+        return;
+      }
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/pages/TagsPage/TagsPage.tsx:129]: Here we can add a condition for early return.
```suggestion
      const fetchCurrentClassificationPermission = async () => {
      if(!currentClassification?.id) return;
    try {
      const response = await getEntityPermission(
        ResourceEntity.CLASSIFICATION,
        currentClassification?.id as string
      );
      setClassificationPermissions(response);
    } catch (error) {
      showErrorToast(error as AxiosError);
    }
  };
```

**Review by Sachin-chaurasiya — APPROVED** [review]

File: openmetadata-ui/src/main/resources/ui/src/pages/ClassificationVersionPage/ClassificationVersionPage.test.tsx (hunk 1 above)
Comment (harsh-vador) [reply openmetadata-ui/src/main/resources/ui/src/pages/ClassificationVersionPage/ClassificationVersionPage.test.tsx:148]: Sure

File: openmetadata-ui/src/main/resources/ui/src/pages/TagsPage/TagsPage.tsx (hunk 3 above)
Comment (harsh-vador) [reply openmetadata-ui/src/main/resources/ui/src/pages/TagsPage/TagsPage.tsx:129]: Sure

File: openmetadata-ui/src/main/resources/ui/src/pages/ClassificationVersionPage/ClassificationVersionPage.test.tsx (hunk 2 above)
Comment (harsh-vador) [reply openmetadata-ui/src/main/resources/ui/src/pages/ClassificationVersionPage/ClassificationVersionPage.test.tsx:167]: Sure

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**